training/mochi-1/args.py

```diff
@@ -0,0 +1,474 @@
+import argparse
```

Comment:
Okay to have a separate file for now for faster iterations. We will be refactoring the repo with a more modular API in the future anyway